src/pages/home/report/ReportActionItemCreated.js

```diff
@@ -47,7 +50,36 @@ const defaultProps = {
     policy: {},
 };
 
+const IMAGE_OFFSET_Y = 75;
+const ANIMATION_BOOST = 1.3;
```

Comment:
After testing, this feels overly-sensitive to me. Let's just get rid of this and not boost the animation at all.